### ISSUE TITLE
Broken on Windows unless you have .NET 4.5+ and PowerShell 3.0 installed

### ISSUE THREAD

**Issue by dcposch** [feross/cross-zip#7]
I had .NET 4.6 installed as well as PowerShell. However, I got the following error:

```
Add-Type : Cannot add type. The assembly 'System.IO.Compression.FileSystem' could not be found.
```

I found lots of posts saying you need .NET 4.5 or greater -- not helpful -- and some dusty old forum post saying you need PowerShell 3. I installed that and now it works.

---

BTW the difference between Windows and Unix here is pretty funny.

Linux and Mac:
- `zip -r swag.zip folder`

Windows:
- Install .NET Framework 4.5 from some 100MB download
- Install Powershell 3.0 as part of "Windows Management Framework 3.0" from 2012
- `powershell.exe -nologo -noprofile -command "& { Add-Type -A 'System.IO.Compression.FileSystem'; [IO.Compression.ZipFile]::CreateFromDirectory('folder', 'wtf.zip'); }"`
- ... or you can do it all out of the box, but only by manual point and click. Ridiculous

**Comment by dcposch** [comment]
Installer: https://www.microsoft.com/en-us/download/details.aspx?id=34595

**Comment by dcposch** [comment]
PS why don't we just use https://www.npmjs.com/package/archiver ?

**Comment by feross** [comment]
I just added information to the readme about the requirements on Windows: https://github.com/feross/cross-zip/commit/8fca4f7f668f0adc68fa67d3ded27d1ac5cb0959

When I searched npm for .zip file creators I couldn't find any that looked reliable and easy to use. None of them accepted a filesystem path to create a zip file from -- they all offered programmatic one directory or file at a time creation. `archiver` looks pretty decent, but using the OS .zip functionality will be more reliable and the installer is super important to get right.
